Muvr/MRAppDelegate.swift

```diff
@@ -66,17 +66,37 @@ class MRAppDelegate: UIResponder, UIApplicationDelegate, MKExerciseModelSource,
     func applicationWillResignActive(application: UIApplication) {
         application.idleTimerDisabled = false
     }
+
+    func loadTextFiles(path bundlePath: String, filename: String, ext: String, separator: NSCharacterSet) -> [String] {
```

Comment:
It doesn't need to be ``internal``, it can be ``private``, or even a nested function in ``getExerciseModel:``